dev-libs/zziplib/zziplib-0.13.59.recipe

```diff
@@ -1,31 +1,95 @@
-DESCRIPTION="
-Lightweight library used to easily extract data from files archived in a \
-single zip file.
-"
+SUMMARY="The zziplib extract data from archives in a single zip"
+DESCRIPTION="Lightweight library used to easily extract data from \
+files archived in a single zip file."
 HOMEPAGE="http://zziplib.sourceforge.net/"
-SOURCE_URI="http://sourceforge.net/projects/zziplib/files/zziplib13/0.13.59/zziplib-0.13.59.tar.bz2/download"
+COPYRIGHT="1999-2010 Guido Draheim"
+LICENSE="GNU LGPL v2.1
+	MPL v1.1"
+SOURCE_URI="https://downloads.sf.net/zziplib/zziplib-$portVersion.tar.bz2"
 REVISION="1"
-STATUS_HAIKU="stable"
-DEPEND=""
-CHECKSUM_MD5="14b5a6fc229afe9916d48358479568d3"
-BUILD()
+CHECKSUM_SSH256="14b5a6fc229afe9916d48358479568d3"
```

Comment:
Yes sorry, as scott said you have two problems here but they somewhat cancel each other.
1) The correct name for the variable is CHECKSUM_*SHA*256
2) The value must be the SHA-256 checksum of the downloaded files. If you did not write it down when haikuporter gave it to you, you have two options:
    a) Go in the recipe directory, delete the download and work directories, and then build the recipe again. It will tell you the expected checksum, you can copy the line directly into the recipe
    b) Go in the recipe directory, and use the ```sha256sum``` tool to compute the SHA-256 checksum of the downloaded file (```sha256sum downloads/zziplib-0.13.59.tar.bz2```)

This checksum identifies the file in an unique way and makes sure people using your recipe are using it on exactly the same sources. Two goals:
1) If they run into problems, they are sure you worked from the same source and it must be something else that is  wrong,
2) Makes sure no one has tampered with the sources to inject a backdoor or some other malicious piece of code.

Since you don't have the right variable name here, haikuporter will jsut ignore your checksum and act as if there wasn't one specified in the recipe. It does not make an error in this case, only a warning. You need to catch it before the build log.